core/libp2p/crypto/crypto_provider.hpp

```diff
@@ -36,35 +36,35 @@ namespace libp2p::crypto {
      * @param data to be encrypted
      * @return encrypted bytes
```

Comment:
```suggestion
     * @return encrypted bytes, if encryption is successful, error otherwise
```
The same for other functions

Reply:
fixed